src/Randomizer.Data/WorldData/Regions/SuperMetroid/Brinstar/GreenBrinstar.cs

```diff
@@ -109,7 +109,7 @@ public MockballHallRoom(GreenBrinstar region, IMetadataService? metadata, Tracke
                     metadata: metadata,
                     trackerState: trackerState);
 
-                TopSuperMissile = new Location(this, 16, 0x8F851E, LocationType.Visible,
+                TopSuperMissile = new Location(this, LocationId.EarlySupersTop, 0x8F851E, LocationType.Visible,
```

Comment:
I wonder if rooms like this would mean that we should maybe include the region at least in the location id? I feel like EarlySupersTop by itself is not very meaningful. GreenBrinstarEarlySupersTop would be more helpful if it's used elsewhere.

Reply:
Yeah, I almost called those spots `MockballHallTop` and `MockballHallBottom`, then was disappointed to see the "official" name of the room is "Early Supers" and not "MockballHall" when the latter seems so much more descriptive. Though I guess there's another "mockball hall" on the way to Ice Beam. Adding `GreenBrinstar` to these two seems like a fine idea to me!

Reply:
Other spots like "The Pit" (Mother Brain's Toilet) could probably have their region in them, too, so I'll be happy to tweak any names that aren't clear enough.

Reply:
Yeah, I think updating all the vague names would be good.

Reply:
Okay, these should look better now